experiments/dynamic-benches/src/network_latency.rs

```diff
@@ -822,16 +808,14 @@ pub mod ppstatic {
         }
 
         fn receive_network(&mut self, msg: NetMessage) -> () {
-            trace!(self.ctx.log(), "Pinger received msg {:?}", msg,);
-            let sender = msg.sender;
-            match msg.data.try_deserialise::<Pong, Pong>() {
+            trace!(self.ctx.log(), "Pinger received msg {:?}", msg, );
+            let sender = msg.sender.clone();
```

Comment:
I'm confused, is this the new NetMessage API? thought clone thingy was not needed.

Reply:
Looks like a wrong merge to me. This is using the old API again.

Reply:
Definitely wrong merge.